python/oneflow/framework/docstr/is_floating_point.py

```diff
@@ -18,7 +18,7 @@
 
 add_docstr(
     oneflow.is_floating_point,
-    r"""Returns True if the data type of input is a floating point data type i.e., one of flow.float64, flow.float32, flow.float16.
+    r"""Returns True if the data type of input is a floating point data type i.e., one of `flow.float64` , `flow.float32` , `flow.float16`.
```

Comment:
```suggestion
    r"""Returns True if the data type of input is a floating point data type i.e., one of `oneflow.float64` , `oneflow.float32` , `oneflow.float16`.
```

我建议这种内嵌代码（而不是整块大段代码）的情况，都使用 `oneflow` 不使用别名 `flow`。用 `flow` 产生的链接，可以正常跳转吗？（我认为不可以）

Reply:
源文档是这么写的，并且 pytorch 文档里一些内嵌的函数并没有实现链接跳转，我们如果需要做到内嵌函数均能跳转的话，还需要专门开辟一个部分（可以加载待完成的文档任务中）找到这些问题再更改，我稍后会加上这一部分